puppet/zulip_ops/files/iptables/header.v4

```diff
@@ -1,14 +1,27 @@
 # This file was auto-generated by Puppet.  Do not edit by hand.
-*filter
 
+# The raw table is used to disable connection tracking for DNS
+# traffic, so it works even when the conntrack table fills.
+*raw
+:PREROUTING ACCEPT [0:0]
+:OUTPUT ACCEPT [0:0]
+-A PREROUTING -p udp -m udp --dport 53 -j CT --notrack
+-A PREROUTING -p udp -m udp --sport 53 -j CT --notrack
+-A OUTPUT -p udp -m udp --dport 53 -j CT --notrack
+-A OUTPUT -p udp -m udp --sport 53 -j CT --notrack
+COMMIT
+
+
+*filter
 # Allow all outbound traffic
 -A OUTPUT -j ACCEPT
 
 # Accept all loopback traffic
 -A INPUT -i lo -j ACCEPT
 
-# Drop all traffic to loopback IPs on other interfaces
--A INPUT ! -i lo -d 127.0.0.0/8 -j DROP
+# Accept incoming traffic related to established connections, or the
+# untracked port-53-UDP set up above
+-A INPUT -m state --state ESTABLISHED,RELATED,UNTRACKED -j ACCEPT
 
 # Accept incoming traffic related to established connections
 -A INPUT -m state --state ESTABLISHED,RELATED -j ACCEPT
```

Comment:
Can you point to the documentation on how the `--state` works? At least one source seems unaware of 
`UNTRACKED` as an option (https://rlworkman.net/howtos/iptables/chunkyhtml/x2702.html), and I'm also guessing maybe this second line isn't needed?

Reply:
Somehow I dropped the wrong two lines -- these should have changed lines 13/14.  Added in a reference to the docs.